### PR TITLE
Move to one bundle

### PR DESCRIPTION
**Pull request by yguedidi**
A prerequisite work to the move to Flex.

- [x] based on #7137

### CODE REVIEW

**Review by j0k3r — COMMENTED** [review]
Why can't this be merged in 2.7?

**Comment by yguedidi** [comment]
> Why can't this be merged in 2.7?

it could if we want! I'd be completely fine with that!
as we said the move to #7115  will be for 2.8 I thought this is similar.

**Comment by j0k3r** [comment]
Moving to one bundle is more _safe_ from my POV than the Flex structure :)

**Comment by yguedidi** [comment]
@j0k3r fine by me then! will rebase and fix tests this morning, I think I found what makes tests fail while working on #7196 , I'll include the hotfix here, except if you prefer that I open a dedicated PR

**Comment by yguedidi** [comment]
@j0k3r @nicosomb @Kdecherf I think it's better to have the upgrade to Symfony 5 merged first and then rebase this one on top of it, hoping that there will be almost no conflict to resolve manually as this PR is the one owning the renaming :)

**Comment by yguedidi** [comment]
@j0k3r @nicosomb @Kdecherf rebased on top of #7137 already so that it's ready to merge after the merge of the upgrade 🙂

File: tests/Wallabag/CoreBundle/WallabagCoreTestCase.php
Comment (j0k3r) [inline tests/Wallabag/CoreBundle/WallabagCoreTestCase.php:56]: The migrate command is creating the schema as well?

Comment (yguedidi) [reply tests/Wallabag/CoreBundle/WallabagCoreTestCase.php:56]: yes, it's is goal. in fact in apps either you use `doctrine:schema:*` commands, either `doctrine:migrations:*` to manage database schema.
see the [very first migration](https://github.com/wallabag/wallabag/blob/master/app/DoctrineMigrations/Version20160401000000.php)

**Review by j0k3r — APPROVED** [review]

**Review by Kdecherf — APPROVED** [review]

**Pull request merged**
